migrate/live-migration/live-migration-from-postgres.md

```diff
@@ -12,402 +12,130 @@ import StepOne from "versionContent/_partials/_migrate_dual_write_step1.mdx";
-
-```sh
-psql -X -d $SOURCE -c 'alter system set old_snapshot_threshold=-1'
-```
-
-Otherwise, if you are using a managed service, use your cloud provider's
-configuration mechanism to set `old_snapshot_threshold` to `-1`.
-
-Next, you should set `wal_level` to `logical` so that the write-ahead log (WAL)
-records information that is needed for logical decoding.
-
-To check the current value of  `wal_level`, run the command:
-
-```sh
-psql -X -d $SOURCE -c 'show wal_level'
-```
-
-If the query returns something other than `logical`, you must change it.
+<LiveMigrationStep2 />
 
-If you have a superuser on a self-hosted database, run the following command:
+## 3. Run the live migration docker image
 
-```sh
-psql -X -d $SOURCE -c 'alter system set wal_level=logical'
-```
-
-Otherwise, if you are using a managed service, use your cloud provider's
-configuration mechanism to set `wal_level` to `logical`.
-
-Restart your database for the changes to take effect, and verify that the
-settings are reflected in your database.
-
-## 3. Set up a replication slot and snapshot
-
-The [replication slot][replication-slot] forms the backbone of the replication
-strategy.
-
-> A slot represents a stream of changes that can be replayed to a client in the
-order they were made on the origin server.
-
-The stream of changes emitted by the slot are buffered into disk until they are
-applied on the target. The instance used to orchestrate the migration (the one
-running the commands) should have enough capacity to store the files, and it
-should be actively monitored to prevent any issues that might result due to
-lack of space.
-
-Before starting, there's an important caveat. To replicate `DELETE` and
-`UPDATE` operations, the source table must either have a primary key or
-`REPLICA IDENTITY` set. Replica identity assists logical decoding in
-identifying the rows being modified. It defaults to using the table's primary
-key.
-
-If a table doesn't have a primary key, you'll have to manually set the replica
-identity. One option is to use a unique, non-partial, non-deferrable index that
-includes only columns marked as `NOT NULL`. This can be set as the replica
-identity:
-
-```sql
-ALTER TABLE {table_name} REPLICA IDENTITY USING INDEX {_index_name}
-```
-
-If there's no primary key or viable unique index to use, you'll have to set
-`REPLICA IDENTITY` to `FULL`. If you are expecting a large number of `UPDATE`
-or `DELETE` operations on the table we **do not recommend** using `FULL`. For
-each `UPDATE` or `DELETE` statement, Postgres will have to read the whole table
-to find all matching rows, which will result in significantly slower
-replication.
-
-```sql
-ALTER TABLE {table_name} REPLICA IDENTITY FULL
-```
-
-Once you're sure all your tables are properly configured for replication, use
-`pgcopydb`'s follow command to create a replication slot:
+First, set the database URIs as environment variables:
```

Comment:
This instruction is too late. The previous steps refer to `$SOURCE`. Also, I would prefer it if we could provide an actual url as an example, instead of an empty string. Compare with the instructions in the note for dual-write and backfill: https://docs.timescale.com/migrate/latest/dual-write-and-backfill/dual-write-from-timescaledb/#3-set-up-schema-and-migrate-relational-data-to-target-database

<img width="632" alt="image" src="https://github.com/timescale/docs/assets/1117749/236fb9e5-62ee-4574-9041-4b82bf283201">